core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/XDDdlParser.scala

```diff
@@ -168,11 +170,27 @@ class XDDdlParser(parseQuery: String => LogicalPlan, xDContext: XDContext) exten
         AddJar(xDContext,jarPath.trim)
     }
 
-  protected lazy val addApp: Parser[LogicalPlan] =
-    (ADD ~> APP ~> ident) ~ (AS ~> ident).? ~ (WITH ~> className) ^^ {
-      case jarPath ~ alias ~ cname =>
-        AddApp(xDContext, jarPath.toString, cname, alias)
-    }
+
+//TODO nvestigate how to detect paths with the parser for add app command
```

Comment:
I would say that the use of quotation marks is a final solution, not a temporary one. Therefore, I would remove this TODO